tests/test_suggestions_model.py

```diff
@@ -0,0 +1,104 @@
+import unittest
+from app import create_app, db
+from app.models import Resource, Suggestion
+from sqlalchemy.exc import IntegrityError
+from datetime import datetime
+import pytz
```

Comment:
nit: pls move this to above `unittest`, since both of these are Python packages and have higher precedence over external packages and this project's imports. Pls also add a newline between `unittest` and `from app import create_app, db` to signify the difference between Python packages and this project's imports.